packages/forma-36-react-components/package.json

```diff
@@ -3,35 +3,68 @@
   "description": "Contentful UI Component Library",
   "version": "3.67.4",
   "main": "dist/index.js",
-  "module": "dist/esm/index.js",
+  "module": "dist/forma-36-react-components.esm.js",
```

Comment:
Wouldn't this break consumer builds who explicitly require an absolute path? if we can avoid by keeping the old convention would be good, if not 🤷‍♂️

Reply:
It would if they require `index`. Have we ever supported this? I'm not sure that we can modify tsdx enough to change such things.